test/10-verify.spec.js

```diff
@@ -705,21 +832,21 @@ for(const [version, mockCredential] of versionedCredentials) {
           should.not.exist(error,
             'Should not throw error when "validFrom" in past');
         });
-        it('should issue "validUntil" in the future', () => {
+        it('should reject "validUntil" in the past', () => {
           const credential = jsonld.clone(mockCredential);
           credential.issuer = 'did:example:12345';
-          credential.validUntil = '2025-10-31T19:21:25Z';
-          const now = '2022-06-30T19:21:25Z';
+          credential.validUntil = '2022-06-31T19:21:25Z';
+          const now = '2025-10-30T19:21:25Z';
           let error;
           try {
             vc._checkCredential({credential, now});
           } catch(e) {
             error = e;
           }
-          should.not.exist(error,
-            'Should not throw error when issuing "validUntil" in future');
+          should.exist(error,
+            'Should throw error when "validUntil" in the past');
         });
-        it('should issue "validUntil" in the past', () => {
+        it('should reject "validUntil" in the future', () => {
```

Comment:
Is this potentially a typo? This shouldn't result in a rejection, right? The assertion specifies `should.not.exist(error)`.